proto/substrait/function.proto

```diff
@@ -78,6 +78,18 @@ message FunctionSignature {
     Type intermediate_type = 13;
 
     repeated Implementation implementations = 15;
+
+    AggregateInvocationType invocation = 16;
+
+    enum AggregateInvocationType {
+      AGGREGATE_INVOCATION_TYPE_UNSPECIFIED = 0;
+
+      // Aggregate function invocation returns all values
+      AGGREGATE_INVOCATION_TYPE_ALL = 1;
+
+      // Aggregate function invocation only returns distinct values
```

Comment:
I feel like the "only returns distinct values" here and below are misleading. The function doesn't return distinct values, it returns a calculation that is based on only distinct values.